app/containers/AudioPlayer/index.tsx

```diff
@@ -124,6 +127,16 @@ const AudioPlayer = ({
 		}
 	}, [paused]);
 
+	useEffect(() => {
+		const unsubscribeFocus = navigation.addListener('focus', () => {
+			audioPlayer.setOnPlaybackStatusUpdate(audioUri.current, onPlaybackStatusUpdate);
+		});
+
+		return () => {
+			unsubscribeFocus();
+		};
+	}, [navigation]);
```

Comment:
Need this to keep the `onPlaybackStatusUpdate` updated according to the audio player component in focus.

ex:

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/121abf5d-bd1f-488b-9bb5-c7349341e31a

- The sound starts to play in the main room. 
- a thread is created on this audio's message
- when we open the thread, the audio pauses (this behavior is expected when open a thread)
- then we can see that the audio component shows the current time the same as the pause of the main room
- then we can play the audio again from where it paused, then stop and back to the main room
- In the main room again, the focus listener calls the function to `setOnPlaybackStatusUpdate` and calls the `onPlaybackStatusUpdate` function of the audio component from the main room and the current time is the expected time from when paused in the thread room
- Also, this approach is a possible solution for when the user go to the Files view and play an audio

Reply:
- Main room <-> thread: audio stops
- Main room <-> files: audio keeps playing